### PR TITLE
🌱 [release-1.5] Update triggers after migration to prow

### PR DESCRIPTION
**Pull request by mboukhalfa**
Update triggers on release 1.5

### CODE REVIEW

**Review by tuminoid — COMMENTED** [review]
/lgtm

File: README.md [hunk 1]
```diff
@@ -132,54 +132,43 @@ To trigger e2e test on a PR, use the following phrases:
...
 
-Release-1.5 branch:
-
-- **/test-ubuntu-e2e-integration-release-1-5** runs integration e2e tests with CAPM3
-  API version v1beta1 and branch release-1.5 on Ubuntu
-- **/test-centos-e2e-integration-release-1-5** runs integration e2e tests with CAPM3
-  API version v1beta1 and branch release-1.5 on CentOS
-
-## Basic tests
+Release-1.6 branch:
```
Comment (kashifest) [inline README.md:135]: wait why should release-1.5 branch has to have anything regarding 1.6

Comment (mboukhalfa) [reply README.md]: thanks done

**Review by Sunnatillo — COMMENTED** [review]
/lgtm

**Pull request merged**